app/src/main/java/com/mailerdaemon/app/clubs/ClubsFragment.java

```diff
@@ -97,6 +98,11 @@ public void getDatabase() {
         }
     }
 
+    @Override
+    public void getData(ShowData showData) {
```

Comment:
Instead of making a new method you can make a new interface for Placement Activity. Or can use default argument in getDatabase(), by changing it to getDatabase(showData:ShowData? = null).

Reply:
This will not affect other files.